pom.xml

```diff
@@ -18,21 +18,54 @@
             <distribution>repo</distribution>
         </license>
     </licenses>
+    <developers>
+        <developer>
+            <name>Manuel Benz</name>
```

Comment:
Nice change. I like that one!

Reply:
Yes, this and the change below are my two favorite changes! :1st_place_medal: